### DISCUSSION TITLE
How to set cpu args?

### DISCUSSION THREAD

**Discussion by avoidik** [lima-vm/lima#1581] [Q&A] [answered]
Hello all,

How can I set additional cpu args? In addition to `qemu64` I would like to enable extra features such as `+sse4.2,+sse4.1,+sse,+sse2,+avx,+avx2`. This way I want to achieve at least `x86-64-v2` level. Please suggest.

I was able to find only a workaround, but is there a better way to do this?

```bash
export QEMU_SYSTEM_X86_64="qemu-system-x86_64 -cpu qemu64,+sse4.2,+sse4.1,+sse,+sse2,+avx,+avx2"
```

**Comment by jandubois** [comment, marked as answer]
In your `lima.yaml` file you should be able to write:

```yaml
cpuType:
  x86_64: qemu64,+sse4.2,+sse4.1,+sse,+sse2,+avx,+avx2
```
